lib/gradle-wrapper.ts

```diff
@@ -1,12 +1,14 @@
 import 'source-map-support/register';
 import { execute } from './sub-process';
 import * as path from 'path';
+import * as os from 'os';
+import {EOL} from 'os';
```

Comment:
Hey, can you fix the formatting here? and maybe use a single import statement, 
`import { EOL, platform } from 'os'`